modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -95,6 +98,7 @@ This procedure creates these tools channels:
 | OS Version | Base Channel                     | Tools Channel
 | {es} 7     | RHEL Expanded Support 7          | RES7-SUSE-Manager-Tools x86_64
 | {es} 8     | RHEL or SLES ES or CentOS 8 Base | RES8-Manager-Tools-Pool for x86_64 and RES8-Manager-Tools-Updates for x86_64
+| {es} 9     | RHEL or SLES ES                  | RES9-Manager-Tools-Pool for x86_64 and RES9-Manager-Tools-Updates for x86_64
```

Comment:
according tho https://github.com/SUSE/spacewalk/issues/19085#issuecomment-1318541475 the base channel will be named "RHEL, Liberty and Clones Base" right now its still called "EL 9 Base"

Reply:
The description should probably also be adjusted to reflect the changed naming

Reply:
ok, I'll now make use of the future name with Liberty.